src/app/shared/ethstats.service.ts

```diff
@@ -1,21 +1,22 @@
 import { Injectable } from '@angular/core'
-import { Observable, of, EMPTY } from 'rxjs'
+import { Observable, of } from 'rxjs'
 import { mergeMap, share } from 'rxjs/operators'
 import * as io from 'socket.io-client'
 
 import { environment } from 'src/environments/environment'
 
-import { EthstatsNode, EthstatsCharts } from './store/ethstats'
+import { EthstatsCharts, EthstatsNode } from './store/ethstats'
+import { Events } from '@celo/celostats-server/src/server/server/Events'
 
 // todo: Sebastian: i tried a long time removing this and i failed. Please make it more obvious
 export interface EthstatsServiceDataNode {
-  action: 'init' | 'add' | 'block' | 'pending' | 'stats'
+  action: Events.Init | Events.Add | Events.Block | Events.Pending | Events.Stats
```

Comment:
If Events is a enum, probably is enught using `actions: Events`, but if we just want to use some of them, is ok

Reply:
I've seen the next interface, makes sense to differentiate it :+1: